### PR TITLE
Extend import order linting to all of galaxy-lib w/fixes.

### PR DESCRIPTION
**Pull request by jmchilton**
Includes a bit more linting of galaxy.util stuff not n galaxy-lib as well.

### CODE REVIEW

File: lib/galaxy/tools/parser/yaml.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 
 from galaxy.tools.deps import requirements
-from .output_collection_def import dataset_collector_descriptions_from_list
-from .output_actions import ToolOutputActionGroup
 from galaxy.util.odict import odict
+
+from .interface import InputSource
+from .interface import ToolSource
+from .interface import PageSource
+from .interface import PagesSource
```
Comment (nsoranzo) [inline lib/galaxy/tools/parser/yaml.py:7]: These are not sorted alphabetically, if you don't collect all imports from the same path in a single command, the flake8 plugin doesn't perform the check apparently.

File: lib/galaxy/util/topsort.py [hunk 2]
```diff
@@ -36,14 +36,16 @@
...
-from galaxy.util.odict import odict as OrderedDict
+
 from six import PY3
 if PY3:
     def list_filter(f, lst):
         return list(filter(f, lst))
 else:
     list_filter = filter
 
+from galaxy.util.odict import odict as OrderedDict
```
Comment (nsoranzo) [inline lib/galaxy/util/topsort.py]: This should be just after 'from six import PY3'

**Comment by jmchilton** [comment]
@nsoranzo - thanks, I have made these changes.

**Comment by nsoranzo** [comment]
:+1: Thanks @jmchilton, great work!

**Comment by martenson** [comment]
this is in conflict now, could you please rebase/merge @jmchilton ? I will merge promptly.

**Comment by jmchilton** [comment]
@martenson Done and tests pass. Things will need to be refactored to move back into galaxy-lib but this is a good step forward. A merge would be appreciated since this will stale quickly.

**Pull request merged**